packages/ui/src/form.tsx

```diff
@@ -187,6 +187,17 @@ const FormMessage = React.forwardRef<
 });
 FormMessage.displayName = "FormMessage";
 
+const FormRootMessage = () => {
+  const { formState } = useFormContext();
+  const error = formState.errors.root;
+
+  if (!error) return null;
+
+  return (
+    <p className="text-sm font-medium text-destructive">{error.message}</p>
+  );
+};
```

Comment:
```suggestion
const FormRootError = React.forwardRef<
  HTMLParagraphElement,
  React.HTMLAttributes<HTMLParagraphElement>
>(({ className, ...props }, ref) => {
  const { errors } = useFormState()
  const rootError = errors.root
  if (!rootError) {
    return null
  }
  return (
    <p
      ref={ref}
      className={cn("text-destructive text-sm font-medium", className)}
      {...props}
    >
      {rootError.message}
    </p>
  )
})
FormRootError.displayName = "FormRootError"
```

...

Reply:
use react.fc, follow the patterns of other components

Reply:
🤦